app/controllers/static_pages_controller.rb

```diff
@@ -7,6 +7,27 @@ def offer
   end
 
   def kontakt
+    @message = Message.new
+  end
+
+  def new
+    @message = Message.new
+  end
+
+  def create
+    @message = Message.new(message_params)
+    if @message.valid?
+      MailerNotifierMailer.mail_to_me(@message).deliver
```

Comment:
1) remove `kontakt` and `new` actions
2) remove Message model
3) change this line to MailerNotifierMailer.mail_to_me(params[:email], params[:name], params[:content]).deliver
4) change mail form url to this actions with :email, :name and :content parameters
5) you can change this action name to contact, as `create` in static_pages controller is misleading